### PR TITLE
feat: add indication for claimable icx from unstaking sicx

### PR DESCRIPTION
**Pull request by hetfly**

### CODE REVIEW

**Comment by parrot9design** [comment]
These animations are delightful. 😍 Nicely done, @hetfly!

**Comment by BennyOptions** [comment]
If @parrot9design approves, so do I!

**Comment by WeiLi512** [comment]
I don't have any unstaking ICX. Is it right to show the indicator in that situation? @parrot9design

image

**Comment by BennyOptions** [comment]
I would think it is not correct, good catch @WeiLi512

**Comment by BennyOptions** [comment]
Ah, looks like it was still a wip, my bad, I saw parrot's comment and assumed it was done

**Comment by parrot9design** [comment]
Haha you guys are getting ahead of yourselves. @hetfly marked it as `wip`, so I think only the UI is done so far, not the logic.

**Comment by WeiLi512** [comment]
haha

**Comment by hetfly** [comment]
You can review now, but I love the enthusiasm! 😄 
@parrot9design @WeiLi512

**Review by parrot9design — APPROVED** [review]
Very happy with this, thanks @hetfly!

File: src/app/components/home/WalletPanel.tsx [hunk 1]
```diff
@@ -162,12 +183,86 @@ const HeaderText = styled(Typography)`
...
+  &:last-of-type {
+    padding-right: 25px;
+  }
 `;
 
 const DataText = styled(Typography)`
   font-size: 16px;
 `;
 
+export const notificationCSS = css`
```
Comment (WeiLi512) [inline src/app/components/home/WalletPanel.tsx:196]: I'd like to move the `notificationCSS` in utils.ts.

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

File: src/app/components/home/WalletPanel.tsx (hunk 1 above)
Comment (hetfly) [reply src/app/components/home/WalletPanel.tsx]: done

**Review by WeiLi512 — APPROVED** [review]

**Pull request merged**